autode/species/species.py

```diff
@@ -1198,5 +1192,60 @@ def explicitly_solvate(self,
         self.solvent.randomise_around(self)
         return None
 
+    def calc_hessian(self,
+                     method:                  ElectronicStructureMethod,
+                     keywords:                Union[Sequence[str], str, None] = None,
+                     numerical:               bool = False,
+                     use_central_differences: bool = False,
+                     num_delta:               Union[float, val.Distance] = val.Distance(2E-3, units='Å')
```

Comment:
Not a good name - sounds like it's number of something. Suggestions?

Reply:
`cartesian_coordinate_shift` ?

Reply:
Much better 👍